BootloaderCorePkg/Tools/ConfigEditor.py

```diff
@@ -793,6 +795,44 @@ def load_from_delta(self):
         self.load_delta_file (path)
 
     def load_delta_file (self, path):
+        # mapping from platform_name to board_name
+        platform_board_name_map = {
+            # ADL series
+            'UPXi12'   : 'adlp',
+            'AdlNCrb'  : 'adln',
+            'AdlNLp5'  : 'adln',
+            'AdlPDdr5' : 'adlp',
+            'AdlPLp4'  : 'adlp',
+            'AdlPLp5'  : 'adlp',
+            'AdlPsCrb' : 'adlps',
+            'AdlPsRvp' : 'adlps',
+            'ADL_S'    : 'adls',
+            'AdlSSdm4' : 'adls',
+            'AdlSSdm5' : 'adls',
+            # RPL series
+            'RPLS S14' : 'rpls',
+            'RPLS S17' : 'rpls',
+            'RPLP CRB' : 'rplp',
+            'RPLPDDR5' : 'rplp',
+            'RPLP LP5' : 'rplp',
+            'RplPsCrb' : 'rplps',
+            'RplPsRvp' : 'rplps'
+        }
```

Comment:
ConfigEditor.py is a common tool. It is not expected to add platform name info. We need think about other solution.

Reply:
Got it, this patch is specific to the use case when user opens the "raw" yaml file under Platform\AlderlakeBoardPkg\CfgData. We may expect user to open the "cooked" yaml file under Build\BootloaderCorePkg\DEBUG_VS2015x86\FV, which is also the one included in our release package, then user won't encounter the issue.
If we have separate CfgDataDef.yamls for different platforms as your suggestion, then could also prevent from this issue :).
Will create another pull request soon.